include/irr/asset/IMeshPacker.h

```diff
@@ -0,0 +1,36 @@
+#ifndef __IRR_I_MESH_PACKER_H_INCLUDED__
+#define __IRR_I_MESH_PACKER_H_INCLUDED__
+#include <cstdint>
+#include <optional>
+
+namespace irr
+{
+namespace asset
+{
+//where should I move it?
+struct DrawElementsIndirectCommand_t
```

Comment:
we hate one place in the engine where we have that struct, I think its in one of the driver headers, could be moved to mesh header

Reply:
+1

Reply:
URGENT